website/_layouts/default.html

```diff
@@ -16,10 +16,10 @@
         </button>
         <div class="collapse navbar-collapse" id="navbarNav">
             <ul class="navbar-nav ms-auto">
-                <li class="nav-item"><a class="nav-link" href="/docs">Docs</a></li>
-                <li class="nav-item"><a class="nav-link" href="/how-tos">How-Tos</a></li>
-                <li class="nav-item"><a class="nav-link" href="/about">About</a></li>
-                <li class="nav-item"><a class="nav-link" href="/community">Community</a></li>
+                <li class="nav-item"><a class="nav-link" href="{{ '/docs' | relative_url }}">Docs</a></li>
```

Comment:
Try using `{{site.baseurl}}/` instead, e.g., `...href="{{site.baseurl}}/docs"...`, etc. This eliminates the need for the `|`.